addons/website_quote/static/src/css/website_quotation.css

```diff
@@ -42,7 +42,10 @@
   padding-left: 30px;
   font-size: 90%;
 }
-
+.o_website_quote .quote-state{
+    font-size:23.4px;
```

Comment:
that size came from computed style as I moved quoatation.state field from the `<h1>` I wanted to keep it's size as it is so applied that style.